pkg/controller/volume/persistentvolume/pv_controller_base.go

```diff
@@ -89,7 +89,7 @@ func NewController(p ControllerParameters) (*PersistentVolumeController, error)
 		volumeQueue:                   workqueue.NewNamed("volumes"),
 	}
 
-	if err := controller.volumePluginMgr.InitPlugins(p.VolumePlugins, controller); err != nil {
+	if err := controller.volumePluginMgr.InitPlugins(p.VolumePlugins, nil, controller); err != nil {
```

Comment:
Why is prober nil? 

Also wondering if you can do `volumePluginMgr.InitPlugins(...)` at layer above and have both PV/PVC controller and attach/detach controller share a volumePluginMgr. <- Don't worry about this for this PR, can do a separate refactor PR 
`